### PR TITLE
publishing march event recap

### PR DESCRIPTION
**Pull request by bennyvasquez**

### CODE REVIEW

File: content/blog/2024-03-26-AlmaLinux-March-Events-roundup.md [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+date: 2024-03-26
+images:
+  - /blog-images/2024/march24events.png
+post: 
+    title: "SCaLE 21x, Hackathon, AlmaLinux Day, and CloudFest!"
+    image: /blog-images/2024/march24events.png
+---
+
+
+March has been a whirlwind and in the last two weeks our event teams visited two sides of the world! In March we saw both Pasedena, California and Rust, Germany! Let's talk about what we loved about each event.
```
Comment (codyro) [inline content/blog/2024-03-26-AlmaLinux-March-Events-roundup.md:17]: Grammar tweaks/Pasadena 

> March has been a whirlwind, and in the last two weeks, our event teams visited two sides of the world! In March, we saw both Pasadena, California, and Rust, Germany! Let’s talk about what we loved about each event.

File: content/blog/2024-03-26-AlmaLinux-March-Events-roundup.md [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+
+-   Deep Dive into User Space: Analyze commonly used software and CMSs to pinpoint potential upgrade obstacles, ensuring smoother transitions
+   -   We tested 3 different real-world scenarios and reported bugs against each scenario.
+-   Focus on Industry Drivers: Identify key elements crucial to data consistency and other industry needs to facilitate necessary upgrades.
+   -   We identified and submitted patches for a number of drivers that could be easily and safely re-added to the kernel, allowing users a chance to continue using unsupported hardware (keep an eye out for the AlmaLinux 8 and 9 betas next month!)
+-   Expand ELevate Support: Extend the capabilities for in-place upgrades across enterprise Linux distributions, including critical updates from CentOS 6 to 7.
+   -   We completed enough research to determine that it's possible and look forward to trying to implement the research over the coming months.
+
+Here are the slides from our [project reporting in PDF format](https://www.dropbox.com/scl/fi/qkokxhum5xsy2pade8qb5/EasingOSUpgrades-cfhack-2024.pdf?rlkey=lb7otq7euorbyj8s2ozo5czo3&dl=0), in case folks would like to look at the details related to our work.
```
Comment (codyro) [inline content/blog/2024-03-26-AlmaLinux-March-Events-roundup.md:40]: Thoughts on uploading this so we serve it as a static asset/it doesn't go poof `N` years from now? 

PTSD from the missing Jack video.

**Review by codyro — CHANGES_REQUESTED** [review]
Also, we should update the banner on the site to point to this new post :).

File: content/blog/2024-03-26-AlmaLinux-March-Events-roundup.md (hunk 2 above)
Comment (bennyvasquez) [reply content/blog/2024-03-26-AlmaLinux-March-Events-roundup.md:40]: good. catch.

**Comment by bennyvasquez** [comment]
Okayed outside of the Github workflows. Merging without GH approval recorded.

**Pull request merged**